app-shells/smrsh/smrsh-8.18.1.ebuild

```diff
@@ -26,6 +26,15 @@ DEPEND="${RDEPEND}
 	sys-devel/m4"
 
 src_prepare() {
+	local confENVDEF="-DXDEBUG=0"
```

Comment:
Isn't this a runtime change (=> needs a new revision)? Or was it off before and you're just being safe by being explicit?

Reply:
Ah, I see, it was in the file before. OK, pop that in the commit message.